MANIFEST.in

```diff
@@ -0,0 +1,19 @@
+include CMakeLists.txt CITATION LICENSE NOTICE .gitmodules mypy.ini requirements.txt version.txt
+recursive-include android *.*
+recursive-include aten *.*
+recursive-include binaries *.*
+recursive-include c10 *.*
+recursive-include caffe2 *.*
+recursive-include cmake *.*
+recursive-include torch *.*
+recursive-include tools *.*
+recursive-include test *.*
+recursive-include docs *.*
+recursive-include ios *.*
+recursive-include third_party *
+recursive-include test *.*
+recursive-include benchmarks *.*
+recursive-include scripts *.*
+recursive-include mypy_plugins *.*
+recursive-include modules *.*
+global-exclude *.o *.so *.dylib *.a .git
```

Comment:
Maybe we should add more global-exclude file patterns similar to SciPy, such as `*.swp`, `*.pyc`? https://github.com/scipy/scipy/blob/master/MANIFEST.in#L31